RabbitMQ.Stream.Client/RawProducer.cs

```diff
@@ -186,7 +188,7 @@ private async Task SemaphoreWait()
                 // Nope, we have maxed our In-Flight messages, let's asynchronously wait for confirms
                 if (!await _semaphore.WaitAsync(TimeSpan.FromSeconds(1)).ConfigureAwait(false))
                 {
-                    LogEventSource.Log.LogWarning("Semaphore Wait timeout during publishing.");
+                    _logger?.LogWarning("Semaphore Wait timeout during publishing");
```

Comment:
Maybe clarify what it means in the message? As in, `"Semaphore Wait timeout during publishing. Server did not respond in time."`?

Reply:
Please go right ahead and make whatever change you think makes sense.